src/main/kotlin/snyk/common/lsp/LanguageServerSettings.kt

```diff
@@ -42,6 +42,7 @@ data class LanguageServerSettings(
     @SerializedName("enableSnykOSSQuickFixCodeActions") val enableSnykOSSQuickFixCodeActions: String? = null,
     @SerializedName("requiredProtocolVersion") val requiredProtocolVersion: String =
         pluginSettings().requiredLsProtocolVersion.toString(),
+    @SerializedName("enableDeltaFindings") val enableDeltaFindings: String = (pluginSettings().netNewIssues == "Net new issues").toString()
```

Comment:
Here, readability could be improved by extracting the comparison into a method `isDeltaFindingsEnabled`

Reply:
I think the value from Snyk Language Server is a string with either true or false. I don't think using a boolean here would work?

Reply:
Fixed!